api/.env.example

```diff
@@ -90,6 +90,8 @@ GCNOTIFY_TEMPLATE_TEST_EMAIL=
 GCNOTIFY_TEMPLATE_TEST_SMS=
 GCNOTIFY_TEMPLATE_TEST_BULK_EMAIL=
 GCNOTIFY_TEMPLATE_TEST_BULK_SMS=
+GCNOTIFY_TEMPLATE_APPLICATION_DEADLINE_APPROACHING_EN=
+GCNOTIFY_TEMPLATE_APPLICATION_DEADLINE_APPROACHING_FR=
```

Comment:
If we're going to be making two of these for every new notification this is going to get out of hand in a hurry. I don't know what the solution is, though.

Reply:
I checked out the GCNotify documentation. This is what how they expect us to use it.
Other ways I could think of is to make the body text also the placeholder/ message variable. 
Option 1: Define a message builder that can build different body of the messages based on what you want to send. Each message will re-use same template.
Option 2:  Re-use the template as much as we can. Like all application related messages one template and try to be as generic as possible and try to squeeze in deadline warning, status change updates to in single template. 
Anything like Password change could be diff template.